src/backend/distributed/commands/utility_hook.c

```diff
@@ -566,12 +566,12 @@ ProcessUtilityInternal(PlannedStmt *pstmt,
 		 * deserialize calls for the statement portable to other postgres servers, the
 		 * workers in our case.
 		 */
```

Comment:
maybe add 1 line of comment: If there are no valid objects, let's skip the qualify and preprocess, and do not diverge from Postgres in terms of error messages.